android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageView.java

```diff
@@ -5,17 +5,97 @@
 
 package org.chromium.chrome.browser.ntp;
 
+import android.os.Bundle;
 import android.content.Context;
 import android.util.AttributeSet;
 import android.view.ViewGroup;
 import android.widget.ImageView;
 import android.widget.TextView;
+import android.content.SharedPreferences;
```

Comment:
there are a lot of imports added here that doesn't appear to be used in this file

Reply:
did you check these? Still looks like a ton of unused imports